ide/languages.hcl/src/org/netbeans/modules/languages/hcl/FontAndColors.xml

```diff
@@ -36,4 +36,8 @@
     <fontcolor name="separator" default="separator"/>
     <fontcolor name="string" default="string"/>
     <fontcolor name="whitespace" default="whitespace"/>
+
+    <fontcolor name="mod-method" default="keyword">
+        <font style="bold"/>
+    </fontcolor>
```

Comment:
I notice this:

```
SEVERE [org.netbeans.modules.csl.editor.semantic.ColoringManager]: no colors for: mod-class with mimetext/x-terraform+x-hcl
```

`mod-class` also needs to be added to the colors definition:

```suggestion
    <fontcolor name="mod-method" default="keyword">
        <font style="bold"/>
    </fontcolor>
    
    <fontcolor name="mod-class" default="keyword">
        <font style="bold"/>
    </fontcolor>
```

Reply:
Played around wit these modifiers, checked in a wrong version. Fixed now.